react-backend/client/src/cb-admin/pages/Dashboard.js

```diff
@@ -17,7 +18,7 @@ const FlexItem = styled.div`
   flex: ${props => props.flex || '1'};
 `;
 
-const ButtonOne = PrimaryButtonNoFill.extend`
```

Comment:
Should this be `Component.extend`?

Reply:
Maybs...will try.

Reply:
This causes a runtime error, so wrapping is the correct way.